site/docs/pages/swap/swap-settings.mdx

```diff
@@ -0,0 +1,378 @@
+---
+title: <SwapSettings /> · OnchainKit
+description: Swap component slippage controls
```

Comment:
Is not just `Slippage`, but can be much more.

Reply:
Also this is a description that reflects users, you want a description that reflects developers.

So what are the keywords devs will use when searching this.

Words like `components` `config` `customize` and so on.

SEO is an art, and every little details matters if you want folks to find your page.

Reply:
Updated to 

> Customizable components for token Swap settings and slippage configuration